docs/user/migration-guide.md

```diff
@@ -0,0 +1,65 @@
+# Migration to v1.0.0-alpha.5
+
+The release includes several breaking changes that should be handled carefully if you are updating Kubeapps from a version prior to v1.0.0-alpha.5. As a summary this release include the following breaking changes:
+
+ - The recommended way of installing Kubeapps is through its helm chart.
+ - The `kubeapps` cli is now deprecated. **It won't be included in future releases**.
+ - Kubeapps no longer includes Tiller, Kubeless nor SealedSecrets by default.
+ - The component `Helm CRD` has been substitued with a secure proxy to the Tiller server. More info about this proxy [here](../../cmd/tiller-proxy/README.md).
+
+These are the steps you need to follow to upgrade Kubeapps to this version.
+
+## Install Tiller
+
+Please follow the steps in [this guide](./securing-kubeapps.md) to install Tiller securely. Don't install the Kubeapps chart yet since it will fail because it will find resources that already exist. Once the new Tiller instance is ready you can migrate the existing releases using the utility command included in `kubeapps` 1.0.0-alpha.5:
+
+```
+$ kubeapps migrate-configmaps-to-secrets
+2018/08/03 16:00:35 Migrated foo.v1 as a secret
+2018/08/03 16:00:35 Done. ConfigMaps are left in the namespace kubeapps to debug possible errors. Please delete them manually
+```
+
+If you list the releases you should be able to see all of them:
+
+```
+$ helm list --tls --tls-ca-cert cert/kubeapps-ca.pem --tls-cert cert/kubeapps.crt --tls-key cert/kubeapps.key
```

Comment:
We can add a note that you can omit the TLS flags if you decide to install Tiller without TLS